### DISCUSSION TITLE
Why are my  material parameters missing many options

### DISCUSSION THREAD

**Discussion by sunsh1nebread** [Anura3D/Anura3D_OpenSource#155] [Q&A]
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/150669478/bd097f76-123e-415f-bf9f-eee9c45dc498)

Hello everybody!
I'm a civil engineering student currently working on my thesis. I compiled the files according to the tutorial, but after importing GID, there were many options missing. I hope someone can help me clarify my doubts. Thank you very much!

**Comment by murphy-john** [comment]
Which material parameters are you looking for in the menu? The menus in the Anura3D problem type in GiD are dynamic. The menu will list only the parameters needed for each material model.
